src/pykiso/connector.py

```diff
@@ -96,28 +100,35 @@ def close(self) -> None:
         with self._lock:
             self._cc_close()
 
-    def cc_send(self, msg: MsgType, raw: bool = False, **kwargs) -> None:
+    def cc_send(self, msg: MsgType, *args, **kwargs) -> None:
         """Send a thread-safe message on the channel and wait for an acknowledgement.
 
         :param msg: message to send
-        :param raw: should the message be converted as pykiso.Message
-            or sent as it is
         :param kwargs: named arguments
         """
+        if ("raw" in kwargs) or args:
```

Comment:
Do we really need to overload the log with this? Is there not a more elegant way here?

Reply:
Indeed this solution is a bit radical, but at least users will definitely adapt their own connectors appropriately as soon as they will integrate the new pykiso release as they'll be spammed with warnings :) 

If we just display a small warning, I'm not sure that it will be annoying enough

Reply:
But we are the one adaption the different aux & connectors right? So the warning is ultimately only for us :)